app/Notifications/RemovedFromServer.php

```diff
@@ -52,6 +52,6 @@ public function toMail(): MailMessage
             ->greeting('Hello ' . $this->user->username . '.')
             ->line('You have been removed as a subuser for the following server.')
             ->line('Server Name: ' . $this->server->name)
-            ->action('Visit Panel', route('index'));
+            ->action('Visit Panel', route('filament.app.resources...index'));
```

Comment:
Use `\App\Filament\App\Resources\ServerResource\Pages\ListServers::getUrl()` instead of route.